xlt-timerrecorder-chrome/src/background.js

```diff
@@ -1,21 +1,60 @@
+// log something so we can see when a service worker was (re-)activated
+log("Service worker activated");
+
+// send a keep-alive message every 10 secs to prevent the service worker from being deactivated after 30 secs of inactivity
+setInterval(
+  () => {
+    log("Sending keep-alive message");
+    send({ action: "KEEP_ALIVE_PING", data: "[]" }, "<dummy>");
```

Comment:
We might adjust function `send` to simply have `messageID` undefined when not given. This would eliminate the neef for the `<dummy>` argument here.

```
function send(data, messageID) {
  const messageString = JSON.stringify({
    messageID: messageID !== undefined ? String(messageID) : undefined,
    data : data
  });
  ... 
}
```

Reply:
I wonder if we need the parameter `messageID` at all. The extension either leaves it undefined or sets only what XLT had passed, however, XLT does not send one either. WDYT about removing this?

Reply:
Indeed, good you've found that! So I better do not change anything here. Will apply your suggestion instead.

Reply:
In fact, XLT requires `messageID` to exist in the JSON message, so we have to set something, even if it is the string "undefined".

Reply:
Well... so be it ;)